backend/clickhouse/traces.go

```diff
@@ -471,7 +471,24 @@ func (client *Client) ReadTracesMetrics(ctx context.Context, projectID int, para
 }
 
 func (client *Client) TracesKeys(ctx context.Context, projectID int, startDate time.Time, endDate time.Time) ([]*modelInputs.QueryKey, error) {
-	return KeysAggregated(ctx, client, TraceKeysTable, projectID, startDate, endDate)
+	traceKeys, err := KeysAggregated(ctx, client, TraceKeysTable, projectID, startDate, endDate)
+	if err != nil {
+		return nil, err
+	}
+
+	defaultTraceKeys := []*modelInputs.QueryKey{}
+	defaultTraceKeys = append(defaultTraceKeys, &modelInputs.QueryKey{
+		Name: "trace_id",
```

Comment:
are there any other keys we may add here? perhaps this `defaultTraceKeys` could be populated in a loop here from some 'defaultTraceKeys' top-level var, like line 45

Reply:
Cleaned up in https://github.com/highlight/highlight/pull/7289/commits/d69c5aa8940709aabecd9e515d67fb4d2b8c7cc9 - good call!